data-serving/scripts/data-pipeline/run_pipeline.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
This is a very well written bash script.

Reply:
This goes above my understanding of bash but I approve

Reply:
you honor and somehow simultaneously shame me